### PR TITLE
Adding QuickStart App for Node.js

### PR DESCRIPTION
**Pull request by rjriel**
Using the same HTML as python I created a Node.js quickstart app.

### CODE REVIEW

**Review by chetvil — APPROVED** [review]

File: node/src/serve.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const { API_PUBLIC_KEY, API_PRODUCT_TYPE } = process.env
+const validProductTypes = ["employment","income"]
+
+if(validProductTypes.indexOf(API_PRODUCT_TYPE) < 0) {
+  console.error("Not a Valid Product Type. Please specify an API_PRODUCT_TYPE of the following:")
+  console.error(validProductTypes)
+  process.exit(-1)
+}
+
+const html = readFileSync(`../html/${API_PRODUCT_TYPE}.html`).toString().replace("{{ public_key }}",API_PUBLIC_KEY).replace("{{ product_type }}",API_PRODUCT_TYPE)
```
Comment (pkuznets) [inline node/src/serve.js:14]: too long line, format it for readability

File: node/src/citadel.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  }
+}
+
+/**
+ * Calls out to Citadel exchanging the public token given by the API request
+ * for an access token to make subsequent requests
+ * @param {String} public_token The token provided by the API request to exchange
+ * @return The access token provided by citadel
+ **/
+const getAccessToken = async (public_token) => {
```
Comment (pkuznets) [inline node/src/citadel.js:24]: calls to citadel API look duplicated, 
it will be better to remove duplication

File: node/src/citadel.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    console.error("Error with /access-tokens/ request")
+    console.error(e)
+    throw e
+  }
+}
+
+/**
+ * Retrieves employment verifications from Citadel
+ * @param {String} access_token The access token provided by Citadel
+ * @return The response from Citadel
```
Comment (pkuznets) [inline node/src/citadel.js:48]: maybe make sense to add some links to documentation to describe what kind of response presented here

File: node/src/citadel.js (hunk 2 above)
Comment (rjriel) [reply node/src/citadel.js:24]: duplicated in what sense?

File: node/src/serve.js (hunk 1 above)
Comment (rjriel) [reply node/src/serve.js]: this is now a multi-line statement. Looks better?

Comment (pkuznets) [reply node/src/serve.js]: yep thx

File: node/src/citadel.js (hunk 2 above)
Comment (pkuznets) [reply node/src/citadel.js:24]: i mean  you can use generic request for different endpoints

```
function request(url, data, errorText) {
  try {
    const response = await fetch(url, {
      method: "POST",
      body: inputBody,
      headers: headers,
    })
    const body = await response.json()
    return body
  } catch (e) {
    console.error(errorText)
    console.error(e)
    throw e
  }

}
```

Comment (rjriel) [reply node/src/citadel.js:24]: refactored per your suggestion

File: node/src/citadel.js (hunk 3 above)
Comment (rjriel) [reply node/src/citadel.js]: Links added to @return lines

**Review by pkuznets — APPROVED** [review]

**Pull request merged**